openapi.yml

```diff
@@ -4212,6 +4212,32 @@ paths:
           type: string
         description: User generated token for Jira authentication.
         required: true
+      - in: query
```

Comment:
why are these query params? it's a put operation, why not have these be in the body of the payload?

Reply:
It should be done the same way as the Bugzilla API key is. Neither in the query nor in the body but in the headers. There was some issue regarding the bindings when the Bugzilla API key was in the body and I think the headers are generally the standard method of transferring the keys.

Reply:
Header is acceptable, but I'd like to know more as to why it's not feasible to have it in the body (perhaps @JakubFrejlach can shed some light).